test/Makefile

```diff
@@ -112,19 +113,28 @@ export DFLAGS=-I$(DRUNTIME_PATH)\import -I$(PHOBOS_PATH)
 export LIB=$(PHOBOS_PATH)
 else
 export ARGS=-inline -release -g -O -fPIC
-export DMD=../src/dmd
+# hack around the fact that on auto-tester MODEL isn't set for the testsuite
```

Comment:
Yes that looks wrong, osmodel.mak will already determine a default MODEL, unless one is explicity specified via `make MODEL=64`. Not sure what fails for you, but you can use `$(error ${MODEL})` to inspect variables.